tests/data/group_system/group_accounts/group_accounts-pam/group_locking_out_password_attempts/rule_accounts_passwords_pam_faillock_deny_root/missing-system-auth.fail.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 # profiles = xccdf_org.ssgproject.content_profile_nist-800-171-cui, xccdf_org.ssgproject.content_profile_ospp
+# remediation = bash
```

Comment:
After some thinking about this, I believe this scenario is completely invalid. We have some rules, where missing config file is finding that is fixable. Usually it's to override some default values.

PAM configurations are not for overriding defaults, but to define behaviour. So in case system-auth is missing, either machine is completely broken and in very unsupported configuration, or not using PAM. We shouldn't try to fix it in such case.